plugins/modules/zos_fetch.py

```diff
@@ -91,27 +88,27 @@
     suboptions:
```

Comment:
I realize we were not consistent with our module options for encode between encoding and fetch modules, ill open a jira to normalize the encodng options to match the fetch module style (i think that is nicer). This is more for an FYI to @asifmahmud

Reply:
@RadhaV2020 
In the encode module we have more details for these options, under the covers they share some similar code , although the options are differently named, i think you can pull from the encoding module and update this one, note this module fetch does not have defaults. 
```
options:
  from_encoding:
    description:
      - The charset of the source I(src).
      - Supported character sets rely on the target version; the most common
        charsets are supported.
    required: false
    type: str
    default: IBM-1047
  to_encoding:
    description:
      - The destination I(dest) charset for the output to be written as.
      - Supported character sets rely on the target version; the most common
        character sets are supported.
    required: false
    type: str
    default: ISO8859-1
```

Reply:
One thing to note here also, is that fetch has a default encoding option, while we decided not have a default for `zos_copy` for example. We might want to have a consistent behavior across these modules

Reply:
@asifmahmud - i took note of your comment, plan to open a JIRA up so we don't let it slip, we can decide what the default should be and then ensure zos_copy adheres to it and if need be change the other modules.

Reply:
@RadhaV2020  - maybe something like:
from:
   description: 
      - The charset of the source I(src).
      - Supported character sets rely on the target version; the most common
        charsets are supported.
  to:
        description:
      - The destination I(dest) charset for the output to be written as.
      - Supported character sets rely on the target version; the most common
        character sets are supported.

I thought it would be reworded a bit differently but as I started to look closer , its really the same for both modules, I don't hink @asifmahmud  would disagree, but he is free to comment. 

The only issue i have with all of this is "the most common character sets are supported."  and i don't have a problem with how its written, this is what i wanted, what i have an issue with is there is over 600 supported encodings, i was thinking to output the list of encodings from `iconv` into an RST file, host it in our web-docs and then link it from the module doc to the web-doc, not sure i can get to it in this beta release, maybe ill get the doc doc and we can link it for our Hub release in the near future.